### PR TITLE
Fix for Audit FAT by constructing BufferManager with early message queue with hard references

### LINKED ISSUE #2194: Fix com.ibm.ws.security.audit.fat.features.DynamicAuditFeatureTest FAT failure where first messages are missing

### PR DESCRIPTION
**Pull request by Channyboy**
#build 
fixes #2194

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

File: dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingQueue.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package com.ibm.ws.logging.internal.impl;
+package com.ibm.ws.collector.manager.buffer;
```
Comment (fmhwong) [inline dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/SimpleRotatingQueue.java]: Please update copyright year.

File: dev/com.ibm.ws.logging_test/test/com/ibm/ws/collector/manager/buffer/SimpleRotatingQueueTest.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-package com.ibm.ws.logging.internal.impl;
+package com.ibm.ws.collector.manager.buffer;
```
Comment (fmhwong) [inline dev/com.ibm.ws.logging_test/test/com/ibm/ws/collector/manager/buffer/SimpleRotatingQueueTest.java]: Please update copyright year.

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
Replaced by #3085

**Pull request closed without merging**